tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -137,6 +182,39 @@ export const getVisibleExecutionDigests = createSelector(
   }
 );
 
+/**
+ * Get the focused alert types (if any) of the execution digests current being
+ * displayed. For each displayed execution digest, there are two possibilities:
+ * - `null` represents no alert.
+ * - An instance of the `AlertType`
+ */
+export const getFocusAlertTypesOfVisibleExecutionDigests = createSelector(
+  selectDebuggerState,
+  (state: DebuggerState): Array<AlertType | null> => {
+    const beginExecutionIndex = state.executions.scrollBeginIndex;
+    const endExecutionIndex =
+      state.executions.scrollBeginIndex + state.executions.displayCount;
+    const alertTypes: Array<AlertType | null> = [];
+    for (let i = beginExecutionIndex; i < endExecutionIndex; ++i) {
+      alertTypes.push(null);
```

Comment:
Can be replaced with `new Array(endExecutionIndex - beginExecutionIndex).fill(null)`.